protoc-gen-openapiv2/internal/genopenapi/template_test.go

```diff
@@ -8583,6 +8598,10 @@ func TestArrayMessageItemsType(t *testing.T) {
 
 		t.Errorf("applyTemplate(%#v).%s = %v want to be %v", file, name, is, want)
 	}
+	d1, _ := json.Marshal(result.Definitions)
+	d2, _ := json.Marshal(expect)
+	fmt.Println(string(d1))
+	fmt.Println(string(d2))
```

Comment:
I think these are for debugging? Can we remove them?

Reply:
done